docs/developers/v2/CONTRACTS_REVIEW.md

```diff
@@ -174,37 +174,10 @@ These posts describe the most important considerations on security reviews.
 - [Sushiswap - Multiple Checks](https://github.com/sushiswap/bentobox/blob/master/documentation/checks.txt)
 - [Not Only Owner - Intro Security Hacking Smart Contracts](https://www.notonlyowner.com/learn/intro-security-hacking-smart-contracts-ethereum)
 
-### Strategies
```

Comment:
@saltyfacu I removed this block because it seemed like a leftover placeholder but lmk if there is a source material that fills it!